### PR TITLE
Fix compilation with 2.071.2.s1 (again)

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**
Soon (TM) it shall be part of CI :)

### CODE REVIEW

File: src/ocean/core/VersionCheck.d
Comment (nemanja-boric-sociomantic) [inline src/ocean/core/VersionCheck.d]: Could this mixin here and the mixing at the line 59 (they are the same) be outside this `version/else`?

Comment (mihails-strasuns-sociomantic) [reply src/ocean/core/VersionCheck.d]: Yep

File: src/ocean/core/VersionCheck.d
Comment (mathias-lang-sociomantic) [inline src/ocean/core/VersionCheck.d:43]: Fixes #99 ?

Comment (mihails-strasuns-sociomantic) [reply src/ocean/core/VersionCheck.d:43]: Yep, linked now.

**Review by mathias-lang-sociomantic — APPROVED** [review]
LGTM if tested

**Review by leandro-lucarella-sociomantic — APPROVED** [review]
LGTM

**Comment by leandro-lucarella-sociomantic** [comment]
Except for the codecov things.

I think we should find a way in which we require all tests to pass before merge (at least on the normal case) or remove codecov if we are going to ignore it anyway.

**Comment by mathias-lang-sociomantic** [comment]
I think as long as https://github.com/sociomantic-tsunami/ocean/issues/204 is not solved we cannot rely on codecov. Also some code will always be untestable by codecov's standard. E.g. it only includes unittests, not integration tests, and we'd need a special tool to merge reports for it to be required.

**Comment by mathias-lang-sociomantic** [comment]
Ping ?

**Comment by mihails-strasuns-sociomantic** [comment]
Sorry, I am on vacation. But will look into this again in a week or two ;)

**Comment by mathias-lang-sociomantic** [comment]
I don't think it requires your attention, just someone to merge. @leandro-lucarella-sociomantic 's concern was about CodeCov, which is orthogonal to this P.R. IMO.

**Comment by nemanja-boric-sociomantic** [comment]
I've just tested this, it sill works with v2.071.2.s1. I'm going to merge this as it LGTM.

**Pull request merged**
